### ISSUE TITLE
pq2hlg.exe produces invalid lut with parameters --max-cll 1000 --lum-scale 1.1

### ISSUE THREAD

**Issue by erazortt** [wswartzendruber/hlg-tools#3]
The following command will produce an invalid lut:
./pq2hlg.exe lut.cube --size 33 --max-cll 1000 --lum-scale 1.1

Version used 1.0.0. However when using 0.4.1 the same command will produce a vaid lut.

These are the first lines of the invalid lut:
```
# Generated by PQ2HLG 1.0.0
# max-cll: 1000
# preview: false
# lum-scale: 1.1
LUT_3D_SIZE 33
0 0 0
0.022888668 0 0
0.04351496 0 0
0.06608266 0 0
0.09117941 0 0
0.11919142 0 0
0.15047367 0 0
0.18539308 0 0
0.22434631 0 0
0.2677706 0 0
0.31615233 0 0
0.37003577 0 0
0.43003172 0 0
0.49682787 0 0
0.5635238 0 0
0.62341523 0 0
0.6787145 0 0
0.73075783 0 0
0.78041124 0 0
0.8282709 0 0
0.87476707 0 0
0.9202232 0 0
0.96489066 0 0
1 0 0
1 0 0
1 0 0
1 0 0
1 0 0
NaN NaN NaN
NaN NaN NaN
NaN NaN NaN
NaN NaN NaN
NaN NaN NaN
0 0.021149805 0
0.020580437 0.020580437 0
0.041949365 0.019404557 0
0.06507446 0.018232668 0
```

**Comment by wswartzendruber** [comment]
In debugging this, I've also discovered that `-t` is used to specify the title **and** the tone mapping method. I'll be fixing that as well.

**Comment by wswartzendruber** [comment]
The problem appears to lie with the BT.2408 tone mapping, regardless of whether `maxrgb` or `rgb` is selected as the type. Essentially, the closer `--lum-scale` gets to 1.0 without actually being 1.0, the more erratic it gets.

**Comment by wswartzendruber** [comment]
This is fixed on my developer machine, but I owe the solution entirely to someone named [Florian Hoech](https://sourceforge.net/u/fhoech/). His work is under a GPLv3 license, and I don't feel right about just ripping it off, even if the answer is in the comments.

I have contacted him about this.

**Comment by wswartzendruber** [comment]
Fixed in 527d6905a7b896bda2e8b2f53f486c8a4ae79b32.
